src/__tests__/element-queries.js

```diff
@@ -193,4 +193,30 @@ test('using jest helpers to check element class names', () => {
   ).toThrowError()
 })
 
+test('test the debug helper prints the dom state here', () => {
+  const Large = `<div>
+        ${Array.from({length: 7000}, (v, key) => key).map(() => {
+          return `<div data-testid="debugging" data-otherid="debugging">
+                Hello World!
+            </div>`
+        })}
+    </div>`
+
+  const {getByText} = render(Large) // render large DOM which exceeds 7000 limit
+  expect(() => expect(getByText('not present')).toBeInTheDOM()).toThrowError()
+
+  const Hello = `<div data-testid="debugging" data-otherid="debugging">
+        Hello World!
+    </div>`
+  const {getByTestId} = render(Hello)
+  process.env.DEBUG_PRINT_LIMIT = 10 // user should see `. . .`
+  expect(() => expect(getByTestId('not present')).toBeInTheDOM()).toThrowError()
```

Comment:
This is a good assertion, but we should add another assertion to make sure the error includes the `...`

You can do this by adding a regex to the `toThrowError` call:

`.toThrowError(/\.\.\./)`

In the one below we should do the opposite


```javascript
.toThrowError(/^((?!\.\.\.).)*$/)
```

I'm _pretty_ sure that should verify that there is no `...` in the error message.